### PR TITLE
Changelog 2.0.0

### PR DESCRIPTION
**Pull request by slemrmartin**
**Gem version: 2.0.0**

**Depends on**:

- [x] #52

### CODE REVIEW

**Comment by lindgrenj6** [comment]
@slemrmartin if you get this rebased I'll merge it :+1:

File: lib/topological_inventory/providers/common/mixins/sources_api.rb [hunk 1]
```diff
@@ -0,0 +1,59 @@
+require "topological_inventory/providers/common/sources_api_client"
+
+module TopologicalInventory
+  module Providers
+    module Common
+      module Mixins
+        module SourcesApi
+          # TODO: Correct?
```
Comment (syncrou) [inline lib/topological_inventory/providers/common/mixins/sources_api.rb]: Can we pull this `#TODO` out?

File: lib/topological_inventory/providers/common/mixins/sources_api.rb [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+          AUTH_NOT_NECESSARY = "n/a".freeze
```
Comment (syncrou) [inline lib/topological_inventory/providers/common/mixins/sources_api.rb]: In regards to the comment above, Is this correct?

Comment (slemrmartin) [reply lib/topological_inventory/providers/common/mixins/sources_api.rb]: @syncrou it was my internal question during refactoring, I forgot to remove it, it'll disappear by rebase :)

**Review by lindgrenj6 — APPROVED** [review]

**Pull request merged**